libraries/AP_Scripting/docs/docs.lua

```diff
@@ -114,6 +114,10 @@ function EFI_State_ud:fuel_consumption_rate_cm3pm(value) end
 ---@param value number
 function EFI_State_ud:fuel_pressure(value) end
 
+-- set field
+---@param value integer
+function EFI_State_ud:fuel_pressure_status(value) end
```

Comment:
```suggestion
---@param status integer
---| '0' # Not supported
---| '1' # Ok
---| '2' # Below nominal
---| '3' # Above nominal
function EFI_State_ud:fuel_pressure_status(status) end
```
We can give scripting users a head start on the enum with in the docs.